src/components/ExplorePage/index.js

```diff
@@ -105,6 +108,9 @@ function ExplorePage({ profile: profileProp, apiUri, ...props }) {
 
   return (
     <div className={classes.root}>
+      <Hidden smDown>
+        <Panel {...panelArgs} {...profileProp} />
```

Comment:
@KobbyMmo `profileProp` does not have updated data per geography, use the state variable profile. https://github.com/CodeForAfrica/PesaYetu/blob/912b9f4575963d03e8d2c6d01a89e5d314c07491/src/components/ExplorePage/index.js#L107 . Because we are de-structuring it here, we could pass geography & data that comes from here